Gemfile.lock

```diff
@@ -37,6 +39,8 @@ PLATFORMS
 
 DEPENDENCIES
   colored
+  minitest
```

Comment:
Gemfile now reflects that minitest is being used for the testing of the application.  Should remove rspec if this is accepted, in a later PR.